.github/workflows/build-pull.yaml

```diff
@@ -8,7 +8,7 @@ on:
     inputs:
       release_tag:
         description: 'Test Build the RPM in pull request.'
-        required: false
+        required: true
```

Comment:
This changes seems to require a Git tag to be specified when manually running the build job, even though it isn't strictly necessary since the version is set to v0.0.0 by default (see "Get version" below). I think we'll want to keep this optional.

On the other hand, this change seems to mismatch the commit message that introduced it: "Force Build to Pass before merge allow". For this, the settings are [in the GitHub repo itself](https://github.com/orgs/community/discussions/75568#discussioncomment-7570257). I don't have access to these settings, but @alexlovelltroy does and he can add a branch rule that forces the build check to pass before merge is allowed.